calico-policies/private-network-isolation/calico-v3/us-east/allow-egress-pods-private.yaml

```diff
@@ -95,6 +95,7 @@ spec:
       - 166.9.22.55/32
       - 166.9.22.8/32
       - 166.9.22.9/32
+      - 166.9.22.216/32
```

Comment:
Same comment as in https://github.com/IBM-Cloud/kube-samples/pull/190/files.  Also it probably makes sense to combine both these changes into one PR